src/queries/monitoring-grant-citation-report.sql

```
@@ -0,0 +1,140 @@
+* - ssdi.reviewOutcomes - string[] - One or more review outcomes
+* - ssdi.reviewTypes - string[] - One or more review types
+* - ssdi.findingReportedDate - date[] - Two dates defining a range for the findingReportedDate to be within. If only one date is supplied, the range is from the supplied date to the current timestamp. If no dates are supplied, this filter is ignored.
+* - ssdi.findingStatuses - string[] - One or more finding statuses
+* - ssdi.findingTypes - string[] - One or more finding types
+* - ssdi.citations - string[] - One or more citations
+* - ssdi.grantNumbers - string[] - One or more grant numbers
+* - ssdi.recipients - string[] - One or more recipient names
+* - ssdi.uei - string[] - One or more UEI values
+*
+* Zero or more SSDI flags can be set within the same transaction as the query is executed.
+* The following is an example of how to set an SSDI flag:
+* SELECT SET_CONFIG('ssdi.reportDeliveryDate', '["2023-01-01", "2023-12-31"]', TRUE);
+*/
+SELECT DISTINCT
+    gr."regionId",
+    gr.number,
+    mr."reportDeliveryDate",
+    mr."reviewType",
+    mf."findingType",
+    ms."citation"
+FROM "Grants" gr
+JOIN "Recipients" r
+    ON gr."recipientId" = r.id
+JOIN "MonitoringReviewGrantees" mrg
+    ON gr.number = mrg."grantNumber"
+JOIN "MonitoringReviews" mr
+    ON mrg."reviewId" = mr."reviewId"
+JOIN "MonitoringReviewStatuses" mrs
+    ON mr."statusId" = mrs."statusId"
+JOIN "MonitoringFindingHistories" mfh
+    ON mr."reviewId" = mfh."reviewId"
+JOIN "MonitoringFindings" mf
+    ON mfh."findingId" = mf."findingId"
+JOIN "MonitoringFindingStandards" mfs
+    ON mf."findingId" = mfs."findingId"
+JOIN "MonitoringStandards" ms
+    ON mfs."standardId" = ms."standardId"
+JOIN "MonitoringFindingStatuses" mfs2
```

I don’t know how I feel about all these being inner joins because it would make stuff silently drop out of the dataset if they ever uploaded dimensional tables without some status. That said, that seems like a very unlikely scenario.